### PR TITLE
add parallelism to the classification training.

### PR DESCRIPTION
**Pull request by exx8**
add parallelism to the classification training.

### CODE REVIEW

**Comment by qq456cvb** [comment]
Hi, have you tested if the performance keeps the same or similar, since dataparallel may affect the batchnorm behavior? See https://discuss.pytorch.org/t/how-does-dataparallel-handels-batch-norm/14040
